src/components/NetworkPanel.vue

```diff
@@ -101,14 +101,20 @@
         </v-hover>
       </v-list-item-group>
     </template>
+
     <div
-      v-else
+      v-if="items.length === 0 && !loading"
       class="ws-detail-empty-list"
     >
       <v-icon color="blue lighten-1">
         info
       </v-icon> There's nothing here yet...
     </div>
+
+    <div v-if="loading">
+      <v-skeleton-loader type="list-item" />
+      <v-skeleton-loader type="list-item" />
+    </div>
```

Comment:
It seems that we should restructure these `v-if`s a bit. Two thoughts here: Vue 2.1.0 has [v-else-if](https://vuejs.org/v2/guide/conditional.html#v-else-if); and we should be able to combine the changed clauses in this section with the big `v-if`ed `<template>` section above.

The overriding case is if we are in a loading state, so the `v-if=loading` case ought to go first. That can be followed up with a `v-else-if=items.length === 0` with the "there's nothing here yet" content, and then a `v-else` that has the big `<template>` section.

This would make this entire section much easier to understand. Let me know what you think.

Reply:
Good catch. I didn't think `v-else-if` was possible. I've cleaned it up in e1aaabd

Reply:
Ah, looks great!